src/GitVersionHelper.php

```diff
@@ -63,4 +63,37 @@ public static function getNameAndVersion()
     {
         return self::appName() . '/' . self::getVersion();
     }
+    
+    /**
+     * Returns comment number with custom length
+     *
+     * @param null $length
+     * @return string comment number
+     *
+     */
+    public static function getCommit($length=null)
```

Comment:
I'd prefer if this were called `getCommitSha` or `getCommitHash` or similar for clarity, since a "commit" is more than just a hash.